.github/github-slack-map.json

```diff
@@ -0,0 +1,86 @@
+{
+  "metamben": "U03AN4TRM6Y",
```

Comment:
My guess is that sorting this list will pay off over time as we manually add and remove people. Perhaps mentioning this in an onboarding guide as well as employee's first PR?

Reply:
good idea!